teachertool/src/transforms/addCriteriaToRubric.ts

```diff
@@ -0,0 +1,39 @@
+import { stateAndDispatch } from "../state";
+import { v4 as uuidV4 } from 'uuid';
+import * as Actions from "../state/actions";
+import { logDebug, logError } from "../services/loggingService";
+
+export async function addCriteriaToRubric(catalogCriteriaIds: string[]) {
+    const { state: teacherTool, dispatch } = stateAndDispatch();
+
+    // Create instances for each of the catalog criteria.
+    const instances = catalogCriteriaIds.reduce((accumulator, catalogCriteriaId) => {
```

Comment:
>reduce [](http://example.com/codeflow?start=41&length=6)

I should probably just use a for...of loop here instead of reduce. Would be cleaner & easier to read.